### DISCUSSION TITLE
On connecting my dapp in trust wallet during connection its not connecting to my custom chain

### DISCUSSION THREAD

**Discussion by vinaywebcom** [WalletConnect/walletconnect-monorepo#4188] [Developer Support]
I have added my custom chain in trust wallet. On connecting my dapp in trust wallet using wallet connect, during connection its not showing my chain, only ethereum chain is there (basically I want to use my custom chain's native currency to interact with smart contract deployed in my custom chain). I am using trustwallet's browser to open my dapp.
Although in PC its working properly with metamask.
I am not sure if its trust wallet's issue or issue with my wallet connect configuration

```
export const Testnet = defineChain({
  id: 00000,
  name: 'XXX',
  nativeCurrency: {
    decimals: 18,
    name: 'XXX,
    symbol: 'XXX',
  },
  rpcUrls: {
    default: { http: ['https://xxx.com/'] },
  },
})
```

This is how I am adding my chain and passing it to 

```
export const config  = defaultWagmiConfig({
    chains: [Testnet], 
    projectId, 
    metadata, 
    ssr: true,
    storage: createStorage({
      storage: cookieStorage,
    })
  })
```

**Comment by glitch-txs** [comment]
that's because it's not supported by Trust

**Reply by vinaywebcom** [reply]
Was not the issue of wallet connect, needed to switch chain from top right menu
